src/editors/numericEditor.js

```diff
@@ -8,23 +7,8 @@ import TextEditor from './textEditor';
  * @dependencies TextEditor numbro
  */
 class NumericEditor extends TextEditor {
-  /**
-   * @param {*} initialValue
-   */
-  beginEditing(initialValue) {
-    if (typeof initialValue === 'undefined' && this.originalValue) {
-      const decimalDelimiter = numbro.cultureData().delimiters.decimal;
-      const numericFormat = this.cellProperties.numericFormat;
-      const cellCulture = numericFormat && numericFormat.culture;
-
-      if (typeof cellCulture !== 'undefined') {
-        numbro.culture(cellCulture);
-      }
-
-      initialValue = (`${this.originalValue}`).replace('.', decimalDelimiter);
-    }
-
-    super.beginEditing(initialValue);
+  beginEditing() {
```

Comment:
Could you add a short comment why you used `this.originalValue` instead of `initialValue`? I think that this information would be very helpful in the future. :)
Additionally, I noticed that `.open()` required a variable `event` as a second argument. It could be a potential issue. What do you think about it?